### PR TITLE
Rm monitor cuda

### PR DESCRIPTION
**Pull request by rmarabini**
Add network, disk IO and GPU monitoring capacities to system_monitor.

### CODE REVIEW

File: pyworkflow/tests/em/protocols/test_protocols_system_monitor.py [hunk 1]
```diff
@@ -53,15 +52,27 @@ def test_pattern(self):
...
+        "doNetwork": True,
+        "netInterfaces": 1,
+        "doDiskIO": True
+        }
         prot2 = self.newProtocol(ProtMonitorSystem, **kwargs)
         prot2.inputProtocols.append(prot1)
         self.launchProtocol(prot2)
 
         baseFn = prot2._getPath(SYSTEM_LOG_SQLITE)
         import os.path
```
Comment (delarosatrevin) [inline pyworkflow/tests/em/protocols/test_protocols_system_monitor.py]: This import should go at the top level.

File: pyworkflow/em/viewer_monitor_summary.py [hunk 2]
```diff
@@ -148,7 +148,8 @@ def _monitorMovieGain(self, e=None):
         MovieGainMonitorPlotter(self.protocol.createMovieGainMonitor()).show()
 
     def _monitorSystem(self, e=None):
-        SystemMonitorPlotter(self.protocol.createSystemMonitor()).show()
+        nifName=self.protocol.nifsNameList[self.protocol.netInterfaces.get()]
```
Comment (delarosatrevin) [inline pyworkflow/em/viewer_monitor_summary.py]: Spaces should be added around =

File: pyworkflow/em/protocol/monitors/protocol_monitor_summary.py [hunk 3]
```diff
@@ -217,7 +244,14 @@ def createSystemMonitor(self):
                                    stdout=True,
                                    cpuAlert=self.cpuAlert.get(),
                                    memAlert=self.memAlert.get(),
-                                   swapAlert=self.swapAlert.get())
+                                   swapAlert=self.swapAlert.get(),
+                                   doGpu=self.doGpu.get(),
+                                   gpusToUse = self.gpusToUse.get(),
```
Comment (delarosatrevin) [inline pyworkflow/em/protocol/monitors/protocol_monitor_summary.py]: Spaces should be removed around =

File: pyworkflow/em/protocol/monitors/protocol_monitor_summary.py [hunk 4]
```diff
@@ -94,6 +96,31 @@ def _defineParams(self, form):
...
+                               '(0,1,2,3, etc). Default is monitor GPU 0 only')
+        group = form.addGroup('NETWORK')
+        group.addParam('doNetwork', params.BooleanParam, default=False,
+                       label="Check Network",
+                       help="Set to true if you want to monitor the Network")
+        group.addParam('netInterfaces', params.EnumParam, choices=self.nifsNameList,
+                      default=1,#usually 0 is the loopback
+                      label="Interface", condition='doNetwork',
+                      help="See http://xmipp.cnb.csic.es/twiki/bin/view/Xmipp/Symmetry"
+                           " for a description of the symmetry groups format in Xmipp.\n"
```
Comment (delarosatrevin) [inline pyworkflow/em/protocol/monitors/protocol_monitor_summary.py]: Are the netInferfaces related to the Xmipp symmetry? This should be a very high-impact method.

Comment (rmarabini) [reply pyworkflow/em/protocol/monitors/protocol_monitor_summary.py]: Hi @delarosatrevin,

   All the above comments have been addressed

**Comment by rmarabini** [comment]
Hi,

  To the best of my knowledge all concerns raised by @delarosatrevin have been addressed

      Roberto

**Comment by delarosatrevin** [comment]
Hi @rmarabini, I'm sorry of being pedantic about keeping a common code style, I truly believe that is a very important thing...specially now that other developers are contributing as well. 

Regarding the spaces around =, in one of my comments was to add them...in another was to remove them. The PEP8 guide states that around = should be spaces, except when using keyword-arguments in the function calls. You have now added (against the convention) spaces to all cases.

I'm sorry again to make you spent time on this. (I'm not sure if in PyCharm or Eclipse there is any tools that help with that)

**Comment by delarosatrevin** [comment]
Thanks @rmarabini. I have tested the changes and the new features of the monitor seems to be working. 
Just a couple of extra comments:
* The help of useGpu params say that a list can be specified comma separated, but it should be separated by spaces
* I find difficult to see the y axis combined of all values (Mb for IO, % for CPU/gpu, etc), but I guess this imply some major changes in the display.

So, there is nothing else from my side and the PR can be merged.

**Comment by rmarabini** [comment]
Let see.

   (1) Regarding pep8 compliance: all files follow the pep8 standrad as defined by the checkout utility "http://pep8online.com"

  (2) Fixed GPU help

  (3) No action has been taken regarding "I find difficult to see the y axis combined of all values (Mb for IO, % for CPU/gpu, etc),". As @delarosatrevin  comments, there is no easy alternative here.

   From may point of view this is ready to go.

**Comment by delarosatrevin** [comment]
PR merged and branch deleted.

**Pull request merged**